src/pages/designing/kits/figma.mdx

```diff
@@ -47,43 +42,76 @@ libraries.
 
 #### 2. Turn on the theme libraries
 
-There are four Carbon themes, two light (White and Gray 10) and two dark (Gray
-90 and Gray 100). Each theme lives in its own Figma library. You can turn on as
-many libraries as you’d like.
+There are four Carbon v10 theme libraries, two light (White and Gray 10) and two
+dark (Gray 90 and Gray 100). Each theme lives in its own Figma library. You can
+turn on as many libraries as you’d like.
+
+If you are using Carbon v11, the v11 Beta White theme library and v11 Text
+styles library are now available. The other v11 theme libraries will be
```

Comment:
I would put the name of the library in a code snippet-it to make it a little easier to read. 

```suggestion
If you are using Carbon v11, the `(v11 BETA) White Theme` library and `v11 Text
Styles` library are now available. The other v11 theme libraries will be
```

Reply:
Agreed! Thank you!